README.md

```diff
@@ -80,15 +87,9 @@ first_order_for_each_user = []
 
 ```
 
-### Answer
-
-```rb
+3.times { |i| puts users[i][:orders][0] 
```

Comment:
This works great, but consider a solution that works with any number of users not just 3